custom_components/variable/sensor.py

```
@@ -319,52 +317,54 @@ async def async_added_to_hass(self):
         """If entity should be polled."""
         return False
 
     @property
-    def force_update(self) -> bool:  # type: ignore[override]
+    def force_update(self) -> bool:
         """Force update status of the entity."""
-        return self._force_update
+        return bool(self._force_update)
 
-    def _update_attr_settings(self, new_attributes=None, just_pop=False):
+    def _update_attr_settings(self, new_attributes: Any = None, just_pop: bool = False) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `new_attributes`

(ANN401)